### PR TITLE
use admin credentials for tsb install operations

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
@sdodson @jim-minter ptal

/assign @sdodson @jim-minter

File: roles/template_service_broker/tasks/install.yml [hunk 1]
```diff
@@ -43,16 +48,18 @@
...
     --param IMAGE="{{ template_service_broker_prefix }}{{ template_service_broker_image_name }}:{{ template_service_broker_version }}"
     --param NODE_SELECTOR={{ template_service_broker_selector | to_json | quote }}
-    | {{ openshift_client_binary }} apply -f -
+    | {{ openshift_client_binary }} apply --config={{ mktemp.stdout }}/admin.kubeconfig -f -
 
 # reconcile with rbac
 - name: Reconcile with RBAC file
   shell: >
-    {{ openshift_client_binary }} process -f "{{ mktemp.stdout }}/{{ __tsb_rbac_file }}" | {{ openshift_client_binary }} auth reconcile -f -
+    {{ openshift_client_binary }} process --config={{ mktemp.stdout }}/admin.kubeconfig -f "{{ mktemp.stdout }}/{{ __tsb_rbac_file }}"
```
Comment (jim-minter) [inline roles/template_service_broker/tasks/install.yml]: @bparees the tox test is failing - wonder if it's the trailing space on this line (change your editor settings? :)

**Comment by spadgett** [comment]
Thanks @bparees. Looks like there are a couple in remove.yml, too

https://github.com/spadgett/openshift-ansible/blob/53ff07d747ccb4e63e750493a3c847ab5434a549/roles/template_service_broker/tasks/remove.yml#L13-L19

**Comment by bparees** [comment]
@jim-minter @sdodson thanks, updated.

**Comment by bparees** [comment]
@sdodson bump

**Pull request merged**